netlify.toml

```diff
@@ -0,0 +1,2 @@
+[context.deploy-preview.environment]
```

Comment:
If I'm not mistaken, this will only work in the deploy preview. To apply to all contexts, including production, this should be something like:
```toml
[build.environment]
  HUGO_VERSION = "0.57.2"
```

Reply:
That is correct but, as promised, I did not want to change your production environment in this branch (at least not without an explicit green light to do so). If you're comfortable with changing the Netflify production environment, I think it would be a smart move since Hugo's latest features are quite useful. :)